src/ios/CDVContact.m

```diff
@@ -1324,6 +1324,7 @@ - (NSObject*)extractPhotos
     NSMutableArray* photos = nil;
 
     if (ABPersonHasImageData(self.record)) {
+        CFIndex photoId = ABRecordGetRecordID(self.record);
```

Comment:
Yes. It can also return `-1` when pemission to address book is denied, but this is hardly ever possible in our case, since we now asking for permissions always (since https://github.com/apache/cordova-plugin-contacts/commit/617fb764bf1c7047cfe02cf346771dd7fa38ed40)